ratik/src/main/java/org/opencadc/ratik/ArtifactValidator.java

```diff
@@ -394,16 +394,16 @@ protected void validateRemote(Artifact remote)
         log.debug("checking explanation 3");
         if (this.remoteSite == null) {
             try {
-                log.debug("starting transaction");
-                this.transactionManager.startTransaction();
-                log.debug("start txn: OK");
+                //log.debug("starting transaction");
+                //this.transactionManager.startTransaction();
```

Comment:
The start and commit of the transaction are commented out?

Reply:
There is only one action so do not need explicit txn.